aws-logs-loggroup/README.md

```diff
@@ -15,3 +15,20 @@ automatically overwritten.
 The code use [Lombok](https://projectlombok.org/), and [you may have to install
 IDE integrations](https://projectlombok.org/) to enable auto-complete for
 Lombok-annotated classes.
+
+## Running Contract Tests
+
+1. Create a KMS CMK for use with CloudWatch Logs (see [the CloudWatch
+    Documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/encrypt-log-data-kms.html)).
+2. Edit `overrides.json` so that every `/KmsKeyId` uses the ARN of that key.
+3. Package the code with Maven (`mvn package`)
+4. Run `sam local start-lambda` in one terminal
+5. Run `cfn test` in another terminal. CFN test will use your credentials to test
+    the resource handlers in your account.
```

Comment:
thoughts on [not explicitly ordering markdown lists](https://guides.github.com/features/mastering-markdown/#syntax) and making these commands [more copy-pasteable](https://github.com/PatMyron/aws-cloudformation-resource-providers-logs/blob/kms-key-id/aws-logs-loggroup/README.md)?

   ```shell
   echo '{
     "CREATE": {
       "/KmsKeyId": "$KEY_ARN"
     }
   }' > overrides.json # so that every /KmsKeyId uses the ARN of that key.
   mvn package # package the code with Maven
   sam local start-lambda # in one terminal
   cfn test # in another terminal. CFN test will use your credentials to test the resource handlers in your account
   ```

Reply:
I disagree with GitHub there, the main goal of markdown is to be readable as a text file :)